routing/index_graph.cpp

```diff
@@ -172,12 +172,17 @@ RouteWeight IndexGraph::GetPenalties(Segment const & u, Segment const & v)
 {
   bool const fromPassThroughAllowed = m_geometry.GetRoad(u.GetFeatureId()).IsPassThroughAllowed();
   bool const toPassThroughAllowed = m_geometry.GetRoad(v.GetFeatureId()).IsPassThroughAllowed();
+  int8_t const passThroughPenalty = fromPassThroughAllowed == toPassThroughAllowed ? 0 : 1;
```

Comment:
Please add a more detailed comment about 0, 1 and penalties.

Reply:
done